src/fides/api/schemas/taxonomy_extensions.py

```diff
@@ -31,7 +31,7 @@ class DataSubject(BaseDataSubject):
 
 class TaxonomyCreateOrUpdateBase(DefaultModel, BaseModel):
     name: Optional[str] = None
-    description: Optional[str]
+    description: Optional[str] = None
```

Comment:
We were missing the `None` so `description` was required and was causing some fixtures to fail